### DISCUSSION TITLE
Bin Group Parameter - uncertain if working or how to initiate.

### DISCUSSION THREAD

**Discussion by Kurtj-hub** [metagenome-atlas/atlas#512] [Q&A] [answered]
Hello, 

When I provide a name for Bingroups in my sample file it does not appear to carry forward through the rest of the pipeline. Should I be specifying these groups somewhere else or have you run into this problem?

What I mean by carry forward is that all of the produced files that I am aware of where bin group is referenced annotates the Bingroup as simply my sample name.

Thanks for any help / information you can provide. If you need any additional information I would be happy to provide it.

**Comment by SilasK** [comment, marked as answer]
Answered #441
